### DISCUSSION TITLE
Dendogram in Python

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#716] [Q&A] [answered]
Apakah ada referensi untuk Hierarchical clustering menggunakan Dendogram, di python ? 

Mohon bantuannya kak, terima kasih 🙏

**Comment by IrfanChairurrachman** [comment]
Dear @yohanespm77 ,

Referensi penerapan Hierarchical Clustering di Python telah cukup banyak referensi dan forum yang membahasanya di internet. Clustering tersebut dapat diterapkan menggunakan `scikit-learn` dan juga `scipy`. Adapun referensinya sebagai berikut:
- [Dokumentasi scikit-learn](https://scikit-learn.org/stable/auto_examples/cluster/plot_agglomerative_dendrogram.html)
- [W3School - Machine Learning - Hierarchical Clustering](https://www.w3schools.com/python/python_ml_hierarchial_clustering.asp)

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.

Regards,
Team Algoritma

**Reply by yohanespm77** [reply]
Terima kasih atas infonya kak. Saya mau bertanya, dalam konsep hierarki ini kalau datanya berupa numerik kan pakai Agglomerative Clustering, kalau datanya berupa kategorik pakainya Divisive hierarchical clustering, kalau datanya berupa campuran numerik dan kategorik pakai metode apa ya kak ?

**Reply by IrfanChairurrachman** [reply, marked as answer]
Mengenai Agglomerative ataupun Divisive itu hanya pendekatan/cara melakukan clustering dan tidak mempengaruhi tipe datanya. Yang mempengaruhi tipe datanya adalah cara menghitung jaraknya. Untuk tipe data campuran dapat menggunakan Gower Distance kemudian dilakukan Agglomerative Clustering. 

Untuk penerapan Agglomerative Clustering menggunakan gower distance dapat melihat referensi notebook berikut: [kaggle notebook](https://www.kaggle.com/code/halflingwizard/clustering-categorical-data-using-gower-distance)

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “Mark as answer” jawaban ini.

Regards,
Team Algoritma
